cmd/gokeyless/gokeyless.go

```diff
@@ -22,55 +27,169 @@ const (
 	defaultEndpoint = "https://api.cloudflare.com/client/v4/certificates/"
 )
 
+// Config represents the gokeyless configuration file.
+type Config struct {
+	LogLevel int `yaml:"loglevel" mapstructure:"loglevel"`
+
+	Hostname     string `yaml:"hostname" mapstructure:"hostname"`
+	ZoneID       string `yaml:"zone_id" mapstructure:"zone_id"`
+	OriginCAKey  string `yaml:"origin_ca_key" mapstructure:"origin_ca_key"`
+	InitEndpoint string `yaml:"init_endpoint" mapstructure:"init_endpoint"`
+
+	CertFile   string `yaml:"cert" mapstructure:"cert"`
+	KeyFile    string `yaml:"key" mapstructure:"key"`
```

Comment:
Where is the mapstructure field tag consumed?

Reply:
The [viper](https://github.com/spf13/viper) package I'm using to handle the config parsing uses mapstructure for parsing.

The `yaml` tags are not actually needed for parsing, but I included them for serializing with the config with the yaml package.